src/test/groovy/unit/org/pih/warehouse/api/ProductApiTest.groovy

```diff
@@ -0,0 +1,117 @@
+package unit.org.pih.warehouse.api
+
+import io.restassured.path.json.JsonPath
+import io.restassured.specification.RequestSpecification
+import org.testng.Assert
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test
+import spock.lang.Shared
+import spock.lang.Specification;
+
+import static io.restassured.RestAssured.*;
+import io.restassured.response.Response;
+
+class ProductApiTest extends Specification {
+
+    @Shared
+    String baseURI = "http://localhost:8080/openboxes";
+
+    @Shared
+    Map headers = [:]
+
+    @Shared
+    boolean setupInitialized = false
+
+    @Shared
+    String productId
+
+    @Shared
+    String locationId = 1
+
+    void setup() {
+        if (!setupInitialized) {
+            RequestSpecification httpRequest = given().formParam("username", "admin").formParam("password", "password");
+            Response response = httpRequest.post(baseURI + "/auth/handleLogin");
```

Comment:
Use the /api/login so you can pass the location ID there.

Reply:
ok